### PR TITLE
move keyserver-related cmds from remote to their own cmd, from sylabs 1905

### PR DESCRIPTION
**Pull request by DrDaveD**
This pulls in sylabs PR

- sylabs/singularity#1905
    which fixed
- sylabs/singularity#1893

The original PR description was:

>   Moves the operations relating to keyservers - adding them, listing them, and removing them - from subcommands of the remote command (singularity remote add-keyserver, singularity remote remove-keyserver, etc.) to their own dedicated command, keyserver (singularity keyserver add, singularity keyserver remove, etc.). See the output of singularity help keyserver for more information.
>
> Also changes the format of the keyserver list output (now generated via singularity keyserver list) in accordance with option (a) outlined in sylabs/singularity#1639

### CODE REVIEW

**Comment by DrDaveD** [comment]
Closing & reopening to kick the DCO test

**Comment by DrDaveD** [comment]
Trying again now that it ran once

File: cmd/internal/cli/remote.go [hunk 1]
```diff
@@ -454,56 +448,24 @@ var RemoteStatusCmd = &cobra.Command{
...
-		}
-
-		if cmd.Flag(remoteKeyserverOrderFlag.Name).Changed && remoteKeyserverOrder == 0 {
-			sylog.Fatalf("order must be > 0")
-		}
-
-		if err := apptainer.RemoteAddKeyserver(name, uri, remoteKeyserverOrder, remoteKeyserverInsecure); err != nil {
-			sylog.Fatalf("%s", err)
-		}
+		sylog.Fatalf("command name changed: use 'keyserver add' instead of 'remote add-keyserver'")
```
Comment (GodloveD) [inline cmd/internal/cli/remote.go:455]: If we are going to leave these command here what is the point of removing the functions and making them error.  Seems like a better idea to leave them in a working state and just add a deprecation warning.  At least until the next minor release (when behavior can change).

**Review by GodloveD — COMMENTED** [review]
Seems like we should probably not change the CLI until the commands have been properly deprecated?

File: cmd/internal/cli/remote.go (hunk 1 above)
Comment (DrDaveD) [reply cmd/internal/cli/remote.go:455]: It was already going to go into the next minor release, not a patch release.  Are you saying it ought to be two minor releases before it gets removed?

Comment (GodloveD) [reply cmd/internal/cli/remote.go:455]: I feel like there ought to be at least one release in which there is a deprecation warning before the behavior changes.  If the next release is a minor one then yes imho this should probably be deprecated with a warning for the entire patch release cycle.

Comment (DrDaveD) [reply cmd/internal/cli/remote.go]: Ok I made that change.

**Review by ikaneshiro — APPROVED** [review]

**Pull request merged**